website/app/templates/patterns.hbs

```diff
@@ -16,8 +16,8 @@
     </p>
     <h2 class="doc-text-h2">Creating your own pattern?</h2>
     <p class="doc-text-body">
-      <LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Design Systems Team</LinkTo>
-      for best practices and suggestions when building and defining local components and patterns.
+      <LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Helios Design Systems Team</LinkTo>
```

Comment:
"Design Systems Team: Use when referring to the team that maintains Helios." [Reference](https://docs.google.com/document/d/1WyoJVpWFVgWbCnZ28WW0gRJwJlj-qV5oY0ExhDR2obs/edit#heading=h.8rc9es8bhfrw)
```suggestion
      <LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Design Systems Team</LinkTo>
```